src/webapi/event.rs

```diff
@@ -332,6 +332,24 @@ reference_boilerplate! {
     convertible to UiEvent
 }
 
+/// The resize event is fired when the document view has been resized.
+///
+/// MDN incorrectly documents this as a UIEvent, but in browsers it is actually
+/// just an Event.
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/Events/resize)
```

Comment:
Did you check what the specs say?

Reply:
It was changed in the specs recently from a UIEvent to an Event to match expectations/browser implementations. Either way, this errors in all browsers if it's specified as a UIEvent rather than an Event.